ibm_ace/tests/test_integration.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .common import skip_windows_ci
+
+pytestmark = [skip_windows_ci, pytest.mark.usefixtures('dd_environment'), pytest.mark.integration]
+
+
+def test(dd_run_check, aggregator, instance, global_tags):
+    from datadog_checks.ibm_ace import IbmAceCheck
+
+    check = IbmAceCheck('ibm_ace', {}, [instance])
+    dd_run_check(check)
+
+    for subscription_type, num_messages in (('resource_statistics', 1), ('message_flows', 2)):
+        tags = [f'subscription:{subscription_type}', *global_tags]
+        aggregator.assert_service_check('ibm_ace.mq.subscription', ServiceCheck.OK, tags=tags)
+        aggregator.assert_metric('ibm_ace.messages.current', num_messages, tags=tags)
+
+    metadata_metrics = get_metadata_metrics()
+    for metric in metadata_metrics:
+        aggregator.assert_metric(metric, at_least=0)
```

Comment:
which metrics are not available in the environment?

Reply:
random handful. tried it?

Reply:
yep!